<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>TV control</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="pushButton_0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="vol_down">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>VOL -</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="ch_up">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CH +</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="vol_up">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>VOL +</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="ch_down">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CH -</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLCDNumber" name="lcdNumber_vol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumber_ch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vol_up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>vol_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vol_down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>vol_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch_up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ch_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch_down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ch_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>push1()</slot>
  <slot>push2()</slot>
  <slot>push3()</slot>
  <slot>push4()</slot>
  <slot>push5()</slot>
  <slot>push6()</slot>
  <slot>push7()</slot>
  <slot>push8()</slot>
  <slot>push9()</slot>
  <slot>push0()</slot>
  <slot>vol_up()</slot>
  <slot>vol_down()</slot>
  <slot>ch_up()</slot>
  <slot>ch_down()</slot>
 </slots>
</ui>
